docs/using-mycroft-ai/basic-commands.md

```diff
@@ -1,104 +1,328 @@
-  issuing commands. Try these to get started.
+  Now that your Mycroft Device is Paired with home.mycroft.ai, you may begin
+  issuing commands. Try these to get started.  
 ---
-
 # Basic Commands
 
-## Default **Skills** installed with Mycroft
-
-### Alarms
-
-> Hey Mycroft, set an alarm for two hours
->
-> Hey Mycroft, set an alarm for 3pm
-
-### Audio record
-
-> Hey Mycroft, record
-
-### Configuration
-
-> Hey Mycroft, configuration update
-
-### DuckDuckGo
-
-> Hey Mycroft, what is Frankenstein?
->
+You issue a command to Mycroft by Speaking the **Wake Word** (e.g. "Hey Mycroft"), awaiting aknowledgement from Mycroft, and then Speaking the command itself (aka the **Utterance**). 
+
+When you issue a command, you *intend* for Mycroft to react in a certain way.  Your **Utterance** contains an **Intent**.  Mycroft uses natural language processing to compare the words in your **Utterance** to expected phrases or keywords as supplied by the installed **Skills**.  When Mycroft determines that there is a *match* between your **Utterance** and a specific expectation (the phrases/keywords), the **Skill** that supplied the matching expectation is notified so that it can respond to your command.
+
+Mycroft comes with a collection of **Skills** installed by default.  Each **Skill** supplies a set of **Intents**, and each **Intent** has a set of expected phrases or keywords.  Each of the default **Skills** is introduced below with a brief description, followed by a sampling of expected **Utternances** (usually grouped by **Intent**). Additional **Skills** may be added to your Mycroft Device; please see [Installing New Skills](https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/installing-new-skills).  
+
+Have fun exploring natural language processing by trying these commands with variations to the wording and phrasing. 
+###
+## The Default **Skills**
+### Alarm - *Manages a collection of alarms.  Weekly repetition is supported.*
+##### Set an alarm
+> Hey Mycroft, wake me in two hours <br>
+> Hey Mycroft, wake me up at 10:23pm <br>
+> Hey Mycroft, set an alarm for two hours <br>
+> Hey Mycroft, set an alarm for 3pm <br>
+> Hey Mycroft, add an alarm for Monday at 8:20 <br>
+> Hey Mycroft, create alarms for every weekday at 6 <br>
+> Hey Mycroft, start a repeating alarm for Fridays at 9:30pm <br>
+> Hey Mycroft, set an alarm *<- - - - - Mycroft asks for time and date*
+##### Snooze the active alarm - *Defaults to 9 minutes*
+> Hey Mycroft, snooze <br>
+> Hey Mycroft, snooze for 15 minutes <br>
+> Hey Mycroft, give me 2 more hours
+##### Delete/stop/cancel/dismiss/remove 
+> Hey Mycroft, stop the alarm <br>
+> Hey Mycroft, delete the Tuesday alarm for 9 <br>
+> Hey Mycroft, remove all alarms
+##### Query alarms
+> Hey Mycroft, when is the next alarm? <br>
+> Hey Mycroft, what are the alarms for today?
+##### Change alarm sound
+> Hey Mycroft, change the sound of the alarm
+###
```

Comment:
Is this to try and create a larger space? 
We should probably leave these as blank lines and let the CSS handle how large of a gap there should be.

Reply:
Yes, I will remove the extra ###.